actionpack/actionpack.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
 
   s.author   = "David Heinemeier Hansson"
   s.email    = "david@loudthinking.com"
-  s.homepage = "http://rubyonrails.org"
+  s.homepage = "https://rubyonrails.org"
```

Comment:
I have added ci skip to this, but will change in gemspec anyways affect anything?

Reply:
Should not add `[ci skip]`, gemspec is definitely not doc, but Ruby code.

Reply:
@kamipo : Removed `[ci skip]` and fixed other comments too.